docs/asciidoc/modules/ROOT/pages/import/gexf.adoc

```diff
@@ -0,0 +1,187 @@
+[[gexf]]
+= Load GEXF (Graph Exchange XML Format)
+:description: This section describes procedures that can be used to import data from GEXF files.
+
+
+
+Many existing applications and data integrations use GEXF to describes a graph with nodes and edges.
+For further information, you should visit the https://gexf.net/[official documentation].
+
+It is possible to load or import nodes and relationship from a GEXF file with the procedures
+ `apoc.load.gexf` and `apoc.import.gexf`. You need to:
+
+* provide a path to a GEXF file
+* provide configuration (optional)
+
+For reading from files you'll have to enable the config option:
+
+----
+apoc.import.file.enabled=true
+----
+
+By default file paths are global, for paths relative to the `import` directory set:
+
+----
+apoc.import.file.use_neo4j_config=true
+----
+
+== Examples for apoc.load.gexf
+
+.load.gexf
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<gexf version="1.2">
+  <graph defaultedgetype="directed">
+    <nodes>
+      <node foo="bar">
+        <attvalues>
+          <attvalue for="0" value="http://gephi.org"/>
+        </attvalues>
+      </node>
+    </nodes>
+  </graph>
+</gexf>
+----
+
+[source, cypher]
+----
+CALL apoc.load.gexf('load.gexf')
+----
+
+.Results
+[opts="header"]
+|===
+| value
+| {_type=gexf, _children=[{_type=graph, defaultedgetype=directed, _children=[{_type=nodes, _children=[{_type=node, _children=[{_type=attvalues, _children=[{_type=attvalue, for=0, value=http://gephi.org}]}], foo=bar}]}]}], version=1.2}
```

Comment:
Sostituisci gli `=` con i `:`, visto che neo4j restituisce il risultato in questo modo, a differenza del toString()